### PR TITLE
Add check token indice in create asset params

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: v3/http/setting_change.go [hunk 1]
```diff
@@ -440,6 +440,11 @@ func getAssetExchange(assets []common.Asset, assetID, exchangeID uint64) (common
 }
 
 func (s *Server) checkCreateAssetParams(createEntry common.CreateAssetEntry) error {
+	if createEntry.Transferable {
+		if err := s.blockchain.CheckTokenIndices(createEntry.Address); err != nil {
```
Comment (secmask) [inline v3/http/setting_change.go]: do we need to make sure `s.blockchain` is not nil?

File: v3/blockchain/flags.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	}
+
+	defaultRateAddress = map[deployment.Deployment]string{
+		deployment.Development: "0x798AbDA6Cc246D0EDbA912092A2a3dBd3d11191B",
+		deployment.Ropsten:     "0x535DE1F5a982c2a896da62790a42723A71c0c12B",
+		deployment.Staging:     "0xe3E415a7a6c287a95DC68a01ff036828073fD2e6",
+		deployment.Production:  "0x798AbDA6Cc246D0EDbA912092A2a3dBd3d11191B",
+	}
+	defaultNodeEndpoint = map[deployment.Deployment]string{
+		deployment.Development: "https://mainnet.infura.io/v3/fd330878eff84d48b97e3023c996dff6",
```
Comment (secmask) [inline v3/blockchain/flags.go]: these values should be put in config because it contain credentials?

Comment (halink0803) [reply v3/blockchain/flags.go]: added check for blockchain != nil.

File: v3/http/setting_change.go (hunk 1 above)
Comment (halink0803) [reply v3/http/setting_change.go]: added check if blockchain == nil.

**Comment by secmask** [comment]
@halink0803 please resolve conflict

**Comment by halink0803** [comment]
> @halink0803 please resolve conflict

Conflict resolved.

File: reservesetting/blockchain/flags.go [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+	}
+
+	defaultRateAddress = map[deployment.Deployment]string{
+		deployment.Development: "0x798AbDA6Cc246D0EDbA912092A2a3dBd3d11191B",
+		deployment.Ropsten:     "0x535DE1F5a982c2a896da62790a42723A71c0c12B",
+		deployment.Staging:     "0xe3E415a7a6c287a95DC68a01ff036828073fD2e6",
+		deployment.Production:  "0x798AbDA6Cc246D0EDbA912092A2a3dBd3d11191B",
+	}
+	defaultNodeEndpoint = map[deployment.Deployment]string{
+		deployment.Development: "https://mainnet.infura.io/",
```
Comment (secmask) [inline reservesetting/blockchain/flags.go]: we can make these value share somewhere and import both by reservesetting and fetcher for better consistent default value.

**Pull request merged**